examples/example.md

```diff
@@ -141,171 +165,176 @@ provider-ionos-cf2fec81b474-54f5d7ddd4-w9w9h        1/1     Running   0
+{% endhint %}
 
-### Create Datacenter, Lan, Postgres Cluster CRs
+### Create datacenter CR, LAN CR, and Postgres cluster CR
 
-Create a Datacenter CR, a Lan CR and a Postgres Cluster CR - using the next command
+To ceate datacenter CR, LAN CR, and Postgres cluster CR, run the following command:
 
 ```bash
 kubectl apply -f examples/ionoscloud/dbaas/postgres-cluster.yaml
 ```
 
-### Get Datacenter, Lan, Postgres Cluster CRs
+### Get datacenter, LAN, and Postgres cluster CRs
 
-Check if the Postgres Cluster CR created is _synced_ and _ready_:
+To check if the Postgres cluster CR created is synced and ready, run the following command:
 
 ```bash
 kubectl get postgresclusters
 ```
 
-Output:
+#### Output
 
 ```bash
 NAME       READY   SYNCED   CLUSTER ID                            STATE      AGE
 example    True    True     9b25ecab-83fe-11ec-8d97-828542a828c7  AVAILABLE  93m
 ```
 
-For more details, use:
+To view more details, run the following command:
 
 ```bash
 kubectl get postgresclusters -o wide
 ```
 
-The external-name of the CR is the Cluster ID from IONOS Cloud. The cluster CR will be marked as ready when the cluster
-is in available state (subject of change).
-
-You can check if the DBaaS Postgres Cluster was created in the IONOS Cloud:
+The ``external-name`` of the CR is the ``Cluster ID`` from IONOS Cloud. The cluster CR will be marked as ``ready`` when the cluster
+is in available state.
 
-- using `ionosctl` (one of the latest [v6 versions](https://github.com/ionos-cloud/ionosctl/releases/tag/v6.1.0)), you
-  can run:
+You can check if the DBaaS Postgres cluster was created in the IONOS Cloud using [<mark style="color:blue;">``ionosctl`` latest versions</mark>](https://github.com/ionos-cloud/ionosctl/releases/tag/v6.1.0). Run the following command:
 
 ```bash
 ionosctl dbaas postgres cluster list
 ```
-
-Output:
+#### Output
 
 ```bash
 ClusterId                              DisplayName   Location   DatacenterId                           LanId   Cidr               Instances   State
 9b25ecab-83fe-11ec-8d97-828542a828c7   testDemo      de/txl     21d8fd28-5d62-43e9-a67b-68e52dac8885   1       192.168.1.100/24   1           AVAILABLE
 ```
 
-- in DCD: go to [DCD Manager](https://dcd.ionos.com/latest/?dbaas=true)
-  to `Manager Resources>Database Manager>Postgres Clusters`
+1. In the **DCD**, go to the **Menu** > **Databases** > **Postgres clusters**.
 
-Check if Datacenter and Lan CRs are created:
+2. Check if the datacenter and LAN CRs are created using:
 
-```bash
-kubectl get datacenters
-kubectl get lans
-```
+  ```bash
+  kubectl get datacenters
+  kubectl get lans
+  ```
 
-### Update Datacenter, Lan, Postgres Cluster CRs
+### Update datacenter, LAN and Postgres ccuster CRs
```

Comment:
Update a Data Center, LAN, and Postgres cluster CRs

Reply:
Done, thank you